ios/FluentUI/IndeterminateProgressBar/IndeterminateProgressBar.swift

```diff
@@ -42,6 +42,7 @@ public struct IndeterminateProgressBar: View, ConfigurableTokenizedControl {
                    maxHeight: height,
                    alignment: .center)
             .background(backgroundColor)
+            .accessibilityAddTraits(.isImage)
```

Comment:
is image correct? if you use just the native activity indicator or uiprogressview, isn't the trait https://developer.apple.com/documentation/uikit/uiaccessibilitytraits/1620187-updatesfrequently ?